### PR TITLE
Specify the websocket server ciphers for iOS9 compatibility

### PR DESCRIPTION
**Pull request by jiayliu**

### CODE REVIEW

File: src/collider/collider/collider.go [hunk 1]
```diff
@@ -46,8 +47,26 @@ func (c *Collider) Run(p int, tls bool) {
 	var e error
 
 	pstr := ":" + strconv.Itoa(p)
-	if tls {
-		e = http.ListenAndServeTLS(pstr, "/cert/cert.pem", "/cert/key.pem", nil)
+	if useTls {
+		config := &tls.Config {
+			MinVersion: tls.VersionTLS11,
```
Comment (tkchin) [inline src/collider/collider/collider.go]: the doc says tls min version should be tls12

**Comment by tkchin** [comment]
lgtm

**Comment by KaptenJansson** [comment]
@jiayliu Did you you deploy this?

**Comment by jiayliu** [comment]
It's already deployed.

On Mon, Nov 9, 2015 at 12:37 AM, Christoffer Jansson <
notifications@github.com> wrote:

> @jiayliu <https://github.com/jiayliu> Did you you deploy this?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/webrtc/apprtc/pull/210#issuecomment-154997176>.
>



-- 
Jiayang Liu

**Comment by KaptenJansson** [comment]
Perfect, thanks!

**Pull request merged**